### PR TITLE
Feat: Creation of the Create Product form

### LINKED ISSUE #91: CREAR UN ESPACIO PARA EL NOMBRE DEL PRODUCTO./ CREATE A SPACE FOR THE PRODUCT NAME.
PROJECT TASK TEMPLATE.

### EXPLANATION

En la vista numero tres de proveedores después de haber seleccionado un producto, debe aparecer un espacio con el nombre del producto seleccionado por el usuario.

In supplier view number three, after selecting a product, a space should appear with the name of the product selected by the user.

### SCREENSHOTS

Image

Image

### CHECKLIST

En esta vista de pantalla numero 3 de proveedores, después de haber seleccionado el producto aparecerá el espacio con el nombre del producto en la parte superior de la pantalla.

In this screen view number 3 of suppliers, after having selected the product, the space with the name of the product will appear at the top of the screen.

### RELATED ISSUE

Taken from the user story: "THE PRODUCT MUST SHOW SOME NECESSARY DATA SUCH AS: NAME, PRICE, DESCRIPTION AND IMAGE"

https://github.com/orgs/Azordev/projects/14/views/1

### LINKED ISSUE #92: CREAR UN ESPACIO PARA EL PRECIO DEL PRODUCTO/ CREATE A SPACE FOR THE PRODUCT PRICE.
PROJECT TASK TEMPLATE

### EXPLANATION

En la vista numero tres de proveedores después de haber ya seleccionado un producto, debe aparecer debajo del producto un espacio donde se pueda ver el precio del producto seleccionado por el usuario.

In supplier view number three, after having selected a product, a space should appear below the product where you can see the price of the product selected by the user.

### SCREENSHOTS

Image

Image

### CHECKLIST

En esta vista de pantalla numero tres de proveedores, después de haber seleccionado el producto, se podrá ver el espacio donde estará el precio del producto.

In this screen view number three of suppliers, after having selected the product, you will be able to see the space where the price of the product will be.

### RELATED ISSUE

Taken from user story "THE PRODUCT MUST SHOW SOME NECESSARY DATA SUCH AS: NAME, PRICE, DESCRIPTION AND IMAGE"

### LINKED ISSUE #93: CREAR UN ESPACIO AMPLIO PARA LA DESCRIPCIÓN DEL PRODUCTO./ CREATE A WIDE SPACE FOR THE PRODUCT DESCRIPTION.
PROJECT TASK TEMPLATE

### EXPLANATION

En la vista numero tres de proveedores después de haber ya seleccionado un producto, debe aparecer un espacio amplio donde se pueda ver la descripción del producto seleccionado por el usuario.

In supplier view number three, after having selected a product, a large space should appear where the description of the product selected by the user can be seen.

### SCREENSHOTS

Image

Image

### CHECKLIST

En esta vista de pantalla numero tres de proveedores después de haber seleccionado el producto, se podrá ver el espacio amplio para la descripción de producto.  

In this screen view number three of suppliers after having selected the product, you will see the large space for the product description.

### RELATED ISSUES

Taken from the user story: "THE PRODUCT MUST SHOW SOME NECESSARY DATA SUCH AS: NAME, PRICE, DESCRIPTION AND IMAGE"

https://github.com/orgs/Azordev/projects/14/views/1

### LINKED ISSUE #94: CREAR UN ESPACIO PARA LA IMAGEN DEL PRODUCTO/ CREATE A SPACE FOR THE IMAGE PRODUCT.
PROJECT TASK TEMPLATE

### EXPLANATION

En la vista numero tres de proveedores después de haber ya seleccionado un producto, debe aparecer el espacio para la imagen del producto seleccionado por el usuario.

In supplier view number three, after having already selected a product, the space for the image of the product selected by the user should appear.

### SCREENSHOTS

Image

Image

### CHECKLIST

Después de haber seleccionado el producto, en la vista numero tres se podrá ver el espacio para la imagen del producto seleccionado.

After selecting the product, in view number three you will see the space for the image of the product selected.

### RELATED ISSUES

Taken from the user story: "THE PRODUCT MUST SHOW SOME NECESSARY DATA SUCH AS: NAME, PRICE, DESCRIPTION AND IMAGE"

https://github.com/orgs/Azordev/projects/14/views/1

### PR DESCRIPTION
**Pull request by luleheco22**
### Description
It was made with the styles shown in the Adobe document

image
image

Please include a summary of the change.

fixes # (#91 #92 #93 #94)

#### Screenshots

If applicable, add screenshots about your changes

#### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### Checklist:

- [ ] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [ ] Doesn't have unnecessary comments or debugging code.
- [ ] The branch is updated with the last dev branch changes.

### CODE REVIEW

**Review by e-azocar — COMMENTED** [review]
Some styles changes login inputs
image

**Review by e-azocar — APPROVED** [review]

**Review by Israel-Laguan — APPROVED** [review]

**Review by Israel-Laguan — APPROVED** [review]

**Review by Israel-Laguan — APPROVED** [review]

**Review by Israel-Laguan — APPROVED** [review]

**Review by Israel-Laguan — APPROVED** [review]

**Comment by e-azocar** [comment]
@luleheco22 please resolve conflicts

File: styles/fonts.scss [hunk 1]
```diff
@@ -60,14 +60,14 @@ $font-base-style: normal;
 $font-base-variant: normal;
 
 
+
```
Comment (Israel-Laguan) [inline styles/fonts.scss]:
```suggestion

```

Erase extra lines

File: styles/fonts.scss [hunk 2]
```diff
@@ -59,10 +59,12 @@ $font-base-weight: 400;
...
 @font-face {
-  font-family: 'Visby';
-  src: url('~/public/assets/fonts/VisbyCF-Regular.otf');
-  font-weight: 400;
+  font-family: 'Visby' serif;
+  src: url('../public/assets/fonts/VisbyRegular.otf');
+  font-weight: 500;
   font-style: normal;
 }
```
Comment (e-azocar) [inline styles/fonts.scss:69]: Please check these Vercel docs to load custom fonts
https://nextjs.org/docs/basic-features/font-optimization
https://solutions-loading-web-fonts.vercel.app/

File: styles/fonts.scss [hunk 3]
```diff
@@ -60,9 +60,9 @@ $font-base-style: normal;
 $font-base-variant: normal;
 
 @font-face {
-  font-family: 'Visby';
-  src: url('~/public/assets/fonts/VisbyCF-Regular.otf');
-  font-weight: 400;
+  font-family: 'Visby' serif;
+  src: url('~/public/assets/fonts/VisbyRegular.otf');
```
Comment (e-azocar) [inline styles/fonts.scss]:
```suggestion
  src: url('/assets/fonts/VisbyRegular.otf');
```
Change font import from `assets` route

Comment (Eghinner) [reply styles/fonts.scss]: my fault. I will solve it. @luleheco22 don't change this file

File: styles/forms.scss [hunk 4]
```diff
@@ -1,39 +1,101 @@
...
+.price {
+  font-size: 19px;
 }
-
 .label {
   display: flex;
   font-family: $font-base-sans-serif-family;
   margin-bottom: 0.5rem;
   margin-top: 1rem;
+  font-family: $font-base-sans-serif-family;
```
Comment (Eghinner) [inline styles/forms.scss:59]: repeated

File: styles/forms.scss [hunk 5]
```diff
@@ -1,39 +1,101 @@
...
   text-align: left;
 }
-
+.input-product {
```
Comment (Eghinner) [inline styles/forms.scss:62]: its the same that `.input-member`. DRY

**Review by Eghinner — APPROVED** [review]

**Comment by Eghinner** [comment]
resolve conflicts

**Comment by angelik0828** [comment]
## ✔️Form validation was performed.

- Visually functional
- Styles added to the fields.
- Change background color
- Added styles to buttons.

image
image

**Review by angelik0828 — APPROVED** [review]

**Pull request merged**